src/gluonts/dataset/split/splitter.py

```diff
@@ -267,7 +257,22 @@ def rolling_split(
 
 class OffsetSplitter(pydantic.BaseModel, AbstractBaseSplitter):
     """
-    Requires uniform data.
+    A splitter that slices training and test data based on an integer offset.
+
+    A positive offset indicates how many observations since the start of each
+    series should be in the training slice; a negative offset indicates how
+    many observations before the end of each series should be excluded from
+    the training slice.
```

Comment:
Maybe it's easier to think about slicing from the left and right to decide what goes into which set.